src/main/java/io/antmedia/muxer/HLSMuxer.java

```diff
@@ -205,7 +205,7 @@ private AVFormatContext getOutputFormatContext() {
 	}
 
 	private boolean isCodecSupported(int codecId) {
-		return (codecId == AV_CODEC_ID_H264 || codecId == AV_CODEC_ID_AAC || codecId == AV_CODEC_ID_MP3 || codecId == AV_CODEC_ID_H265);
+		return (codecId == AV_CODEC_ID_H264 || codecId == AV_CODEC_ID_AAC || codecId == AV_CODEC_ID_MP3 || codecId == AV_CODEC_ID_H265 || codecId == AV_CODEC_ID_PCM_ALAW);
```

Comment:
As far as I know, PCM is not supported in HLS. It seems that this code snippets claim PCM is supported by HLS

Reply:
Got it, it is true, found the root cause and tried to fix it, please check below commit.